sopel/modules/reddit.py

```diff
@@ -189,14 +196,57 @@ def comment_info(bot, trigger, match):
 
 
 # If you change this, you'll have to change some other things...
-@commands('redditor')
-@example('.redditor poem_for_your_sprog')
-def redditor_info(bot, trigger, match=None):
+def subreddit_info(bot, trigger, match, is_command=False):
```

Comment:
Calling the last arg `is_command` instead of `commanded` necessitates some otherwise-meaningless changes later in this function. We're fairly consistent about using the name `commanded` for this purpose in other plugins.